### PR TITLE
Small cleanup of install docs (#2613)

### PR DESCRIPTION
**Pull request by daineAMD**

### CODE REVIEW

File: docs/install/Linux_Install_Guide.rst [hunk 1]
```diff
@@ -122,7 +122,7 @@ CMake has a minimum version requirement listed in the file ``install.sh``. See `
 
 Dependencies are listed in the script ``install.sh``. Passing the ``-d`` flag to ``install.sh`` installs the dependencies.
 
-However, for the test and benchmark clients' host reference BLAS, it is recommended that you manually download and install AMD's ILP64 version of AOCL-BLAS 4.1 or 4.0 from https://www.amd.com/en/developer/aocl.html.
+However, for the test and benchmark clients' host reference BLAS, it is recommended that you manually download and install AMD's ILP64 version of AOCL-BLAS 4.2 from https://www.amd.com/en/developer/aocl.html.
```
Comment (TorreZuk) [inline docs/install/Linux_Install_Guide.rst]: Could you make it  "4.2 gcc" as our CMake only looks for that version for 4.2

**Review by TorreZuk — APPROVED** [review]
LGTM but minor tweak suggested

**Review by amcamd — APPROVED** [review]
It is good to see these updates to correct documentation.

File: docs/install/Linux_Install_Guide.rst [hunk 2]
```diff
@@ -96,14 +96,14 @@ For Centos, use:
...
-There are GitHub branches at the rocBLAS site with names rocm-major.minor.x where major and minor are the same as in the ROCm version. For ROCm version 4.0.0.40000-23, you must use the following to download rocBLAS:
+There are GitHub branches at the rocBLAS site with names rocm-major.minor.x where major and minor are the same as in the ROCm version. To download rocBLAS, you can use the following command:
 
 ::
 
    git clone -b release/rocm-rel-x.y https://github.com/ROCm/rocBLAS.git
    cd rocBLAS
 
-Replace x.y in the above command with the version of ROCm installed on your machine. For example, if you have ROCm 5.0 installed, then replace release/rocm-rel-x.y with release/rocm-rel-5.0.
+Replace x.y in the above command with the version of ROCm installed on your machine. For example, if you have ROCm 6.0 installed, then replace release/rocm-rel-x.y with release/rocm-rel-6.0.
```
Comment (randyh62) [inline docs/install/Linux_Install_Guide.rst]: While we are making changes, why not change the release to 6.2?

**Review by randyh62 — APPROVED** [review]

File: docs/install/Linux_Install_Guide.rst (hunk 2 above)
Comment (daineAMD) [reply docs/install/Linux_Install_Guide.rst]: Sure, done.

**Pull request merged**